index.css

```diff
@@ -518,3 +505,99 @@ h3 {
   line-height: 43px;
   color: #f4f5f7;
 }
+
+.form-section {
+  display: grid;
+  row-gap: 58px;
+  padding: 0 24px 0 24px;
+  color: white;
+}
+
+.wrapper ul {
+  list-style: none;
+  display: flex;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  padding: 24px;
+  width: 200px;
+}
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  gap: 50px;
+}
+
+.form-title {
+  font-family: 'Inter', sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #172b4d;
+}
+
+.form-section form {
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+  justify-content: center;
+}
+
+.btn-contact {
+  background-color: #ff6b00;
+  border: none;
+  color: white;
+  padding: 10px 20px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  margin: 4px 2px;
+  cursor: pointer;
+  border-radius: 5px;
+}
+
+.line3 {
+  height: 5px;
+  width: 134px;
+  border: 1px solid hsla(214, 38%, 8%, 1);
+  background-color: hsla(214, 38%, 8%, 1);
+  border-radius: 10px;
+  align-items: center;
+  justify-self: flex-end;
+}
+
+form input {
+  height: 48px;
+  padding: 5px;
+  border-top: none;
+  border-right: none;
+  border-left: none;
+  border-bottom: 1px solid #dbd8d7;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #979493;
+}
+
+form textarea {
+  padding: 12px;
+  width: 303px;
+  background: #fbf8f7;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+  border: none;
+  outline: none;
+}
```

Comment:
- [ ] Kindly make sure that you use the provided Figma design font and colors. Kindly use the font 'inter' for styling the text area section.

**This is how your text area currently looks like:**
![image](https://user-images.githubusercontent.com/86318284/177399145-ee76e334-54cb-4e20-b216-50dc13417106.png)

**This is the desired Figma design:**
![image](https://user-images.githubusercontent.com/86318284/177399271-180545f2-6a24-4219-ac43-c1089444a0da.png)